### DISCUSSION TITLE
_TZE204_dpqsvdbi tuya unsupported

### DISCUSSION THREAD

**Discussion by YeaYean** [Koenkk/zigbee2mqtt#24027] [General]
A few days ago, I purchased a Zigbee version of curtains, but now when I try to connect it via Zigbee2MQTT (Z2M), it says "Not Supported" after pairing.
I tried to solve this problem until I found this great website, but unfortunately I couldn't find anything in the content of https://github.com/Koenkk/zigbee-herdsman-converters/blob/master that worked for me currently Device model TZE204dpqsvdbi is supported by /src/devices/tuya.ts.   what I can do to connect my device via Zigbee2MQTT?
![image](https://github.com/user-attachments/assets/a5f4c0eb-04ba-4082-b89f-89aa881b4f41)

![image](https://github.com/user-attachments/assets/641005ce-b338-4579-b23d-ae085648ef3c)
![image](https://github.com/user-attachments/assets/8cffb455-31d1-450f-913c-1fce36df59cb)

not found type :(
![image](https://github.com/user-attachments/assets/0c624eeb-6a6f-4378-af0a-a7ccc2731913)

**Comment by YeaYean** [comment]
![image](https://github.com/user-attachments/assets/af0c93a8-4188-4c37-a39e-0f0cca40c5db)
I found that I don't have any functions available. Does this mean I can't control this device through Z2M?

**Comment by YeaYean** [comment]
<img width="1048" alt="截圖 2024-09-21 00 29 57" src="https://github.com/user-attachments/assets/76e34b5f-3746-4e43-9aa1-aa1b221a64ee">
When I press the controller but no status is displayed, does it mean I can only give up...? :(

**Comment by Koenkk** [comment]
Could you try to

- Stop Z2M
- Open `data/database.db` with a text editor and change all occurrences of `_TZE204_dpqsvdbi ` to `_TZE200_udank5zs`
- Start Z2M and see if the device now works

**Reply by YeaYean** [reply]
It does work! Now I can control it on both the HA panel and the Zigbee panel. Do I need to use this configuration now, or is this just for testing purposes?"
<img width="1048" alt="截圖 2024-09-22 20 39 56" src="https://github.com/user-attachments/assets/94cb6c7f-b398-4b1f-8769-787e1be0e4b7">

**Reply by Koenkk** [reply]
Great, added it! You can change it back after updating to the next release (1 October)

**Reply by YeaYean** [reply]
Thanks a lot! I will update to the new version on 10/1. Thanks!
